config/recipes/gclb/README.asciidoc

```diff
@@ -56,3 +56,12 @@ Access Elasticsearch.
 ----
 curl -H "Host: elasticsearch.hulk" --resolve "elasticsearch.hulk:443:$INGRESS_HOST" -k -u "elastic:$ELASTICSEARCH_PASSWORD" 'https://elasticsearch.hulk/_cat/health?v'
 ----
+
```

Comment:
Not added in this PR but should we add a not about the minimal version of cert-manager you are expecting (I had an old one installed and need to upgrade first before this recipe worked ootb)